cropper/src/main/java/com/canhub/cropper/CropImage.kt

```diff
@@ -185,22 +191,26 @@ object CropImage {
     fun getPickImageChooserIntent(
         context: Context,
         title: CharSequence?,
-        includeDocuments: Boolean, // todo, remove this. Should always be false for image to crop.
-        includeCamera: Boolean,
+        options: PickImageContractOptions
     ): Intent {
+        val includeDocuments = options.includeDocuments
+        val includeCamera = options.includeCamera
+        val includeGallery = options.includeGallery
         val allIntents: MutableList<Intent> = ArrayList()
         val packageManager = context.packageManager
         // collect all camera intents if Camera permission is available
         if (!isExplicitCameraPermissionRequired(context) && includeCamera) {
             allIntents.addAll(getCameraIntents(context, packageManager))
         }
-        allIntents.addAll(
-            getGalleryIntents(
-                packageManager,
-                Intent.ACTION_GET_CONTENT,
-                includeDocuments
+        if (includeGallery) {
+            allIntents.addAll(
+                getGalleryIntents(
+                    packageManager,
+                    Intent.ACTION_GET_CONTENT,
+                    includeDocuments
```

Comment:
What if `includeDocuments` is true, but `includeGallery` is false?

Reply:
Great point, I will need to refactor